README.md

```diff
@@ -6,7 +6,7 @@ Shunting Yard reads serialized Hive Metastore Events from a queue ([AWS SQS](htt
 
 You can obtain Shunting Yard from Maven Central:
 
-[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.expediagroup/shunting-yard/badge.svg?subject=com.expediagroup:shunting-yard)](https://maven-badges.herokuapp.com/maven-central/com.expediagroup/shunting-yard) [![Build Status](https://travis-ci.org/ExpediaGroup/shunting-yard.svg?branch=master)](https://travis-ci.org/ExpediaGroup/shunting-yard) [![Coverage Status](https://coveralls.io/repos/github/ExpediaGroup/shunting-yard/badge.svg?branch=master)](https://coveralls.io/github/ExpediaGroup/shunting-yard?branch=master) ![GitHub license](https://img.shields.io/github/license/ExpediaGroup/shunting-yard.svg)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.expediagroup/shunting-yard/badge.svg?subject=com.expediagroup:shunting-yard)](https://maven-badges.herokuapp.com/maven-central/com.expediagroup/shunting-yard) [![Build Status](https://travis-ci.org/ExpediaGroup/shunting-yard.svg?branch=master)](https://travis-ci.org/ExpediaGroup/shunting-yard) ![build](https://github.com/ExpediaGroup/shunting-yard/workflows/build/badge.svg?event=push)  [![Coverage Status](https://coveralls.io/repos/github/ExpediaGroup/shunting-yard/badge.svg?branch=master)](https://coveralls.io/github/ExpediaGroup/shunting-yard?branch=master) ![GitHub license](https://img.shields.io/github/license/ExpediaGroup/shunting-yard.svg)
```

Comment:
There is some mistake if you see it on github.

Reply:
Yes, but I'm 99% sure it's because the file its referring to doesn't exist yet as it is created in this PR (the main.yml file). So if everything goes as it should, when this PR is pushed the file will be created and the badge will be displayed as normal.

Reply:
Yeah, I think it will be OK, let's get this merged in and have the workflow run once and we can check.